components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -38,7 +39,8 @@
  in a sheet from the bottom.
 
  */
-__attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController : UIViewController
+__attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
+    : UIViewController<MDCElevatable, MDCElevationOverriding>
```

Comment:
I think there should be a space between UIViewController and the brackets

Reply:
I don't know what is right, I updated it to have a space and clang got made so I think there shouldn't be if clang didn't get mad I'd have no idea lol